models_generation/gaussian_mixture.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+import pandas as pd
+from sklearn.mixture import GaussianMixture
+from zipfile import ZipFile
+import os
+
+
+MODELS_PATH = os.getenv("MODELS_PATH", "./models")
+
+
+if __name__ == "__main__":
+    print("Training Gaussian Mixture model...")
+    # TODO replace with local file
+    df = pd.read_csv('https://raw.githubusercontent.com/def-au1t/AGH_8_AI-Project/master/out_rpi_fan_25_steps1_baseline.csv')
```

Comment:
Could you change it, so the CSV path would be an (optional) parameter provided by the user?